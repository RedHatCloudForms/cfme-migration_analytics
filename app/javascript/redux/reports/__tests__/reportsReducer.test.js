// TODO add unit tests for reportsReducer.js

test.todo('add unit tests here');
