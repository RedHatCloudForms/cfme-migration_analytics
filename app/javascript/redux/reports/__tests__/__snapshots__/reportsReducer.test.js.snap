// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reports reducer constructs initial state correctly 1`] = `
Object {
  "errorFetchingReports": null,
  "errorFetchingResult": null,
  "errorFetchingTask": null,
  "errorRunningReport": null,
  "fetchingResultHrefs": Array [],
  "fetchingTaskHrefs": Array [],
  "numReportFetchesPending": 0,
  "reportRunsByHref": Object {},
  "reports": Array [],
  "resultsByHref": Object {},
  "runningReportHrefs": Array [],
  "tasksByHref": Object {},
}
`;
